sig-operations/charter.md

```diff
@@ -0,0 +1,38 @@
+# SIG Operations
+
+## Scope
+
+The goal of this SIG is to enable management and operations for hardware and platform tooling within the Operate First Community Cloud (op1st)
+
+### In scope
+
+* Define what is part and not part of the Operate First Cloud platform offering
+* Communicate a platform roadmap
+* Day to day operations of platform
+* Operating and ensuring stability of
```

Comment:
```
* Operating and ensuring stability of
```
Of what? is this a typo or are the items below suppose to be a sublist?

Reply:
yes this is a typo, thanks!

Reply:
fixed